src/app/static/src/tests/components/runWorkflow/runWorkflow.test.ts

```diff
@@ -186,11 +187,11 @@ describe(`runWorkflow`, () => {
         wrapper.find("#create-workflow").trigger("click")
 
         setTimeout(async () => {
-            expect(wrapper.vm.$data.runWorkflowMetadata).toStrictEqual(emptyWorkflowMetadata);
+            expect(wrapper.vm.$data.runWorkflowMetadata).toStrictEqual({...emptyWorkflowMetadata, git_branch: "master"});
```

Comment:
I think this is happening because runWorkflow is now being continually updated to changes in the metadata and the ui selects the first branch by default, which is master.

Reply:
Sounds about right - one of the issues with using `mount` rather than `shallowMount` in so-called unit tests..